public/redesign/index.html

```
@@ -0,0 +1,1752 @@
+═══════════════════════════════════ -->
+    <section class="section" id="docs">
+      <div class="section-label">Section 03</div>
+      <h2 class="section-title">Docs experience</h2>
+      <p class="section-desc">
+        Both product lines share the same doc shell. A product switcher in the sidebar lets
+        developers move between the two. Each sidebar has a persistent cross-sell prompt at the
+        bottom.
+      </p>
+
+      <div class="docs-tabs">
+        <button class="docs-tab active-agents" id="dtab-agents" onclick="showDocsTab('agents')">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)